### ISSUE TITLE
fusion组件主题包配置不生效

### ISSUE THREAD

**Issue by lilithyang2018** [alibaba/ice#6560]
### Describe the bug

参考飞冰文档https://v3.ice.work/docs/guide/advanced/fusion#themepackage，为fusion组件配置主题包
,结果主题样式没有生效 如下图
![fusion button](https://github.com/alibaba/ice/assets/39897706/db6f942f-c95e-4fa1-a6dd-b339a90fd93b)

### Expected behavior

希望能通过配置得到下图的主题样式
![fusion button 2](https://github.com/alibaba/ice/assets/39897706/78a350f5-2bd7-432b-b543-4773fa2ec89e)

### Actual behavior

通过fusion插件配置的主题样式没有对fusion组件生效，
只有在global.css中全量引入css才有效果

### Version of ice.js

3.3.3

### Content of build.json or ice.config.mts

```TypeScript
import { defineConfig } from '@ice/app';
import fusion from '@ice/plugin-fusion';

// The project config, see https://v3.ice.work/docs/guide/basic/config
const minify = process.env.NODE_ENV === 'production' ? 'swc' : false;
export default defineConfig(() => ({
  // Set your configs here.
  minify,
  server: {
    onDemand: true,
    format: 'esm',
  },
  plugins: [
    fusion({
      importStyle: true,
      themePackage: '@alifd/theme-1',
    }),
  ]
}));
```

### Additional context

{
  "name": "@ice/lite-scaffold",
  "version": "1.2.0",
  "description": "A new ice.js project.",
  "dependencies": {
    "@alifd/next": "^1.26.25",
    "@alifd/theme-1": "^0.4.2",
    "@ice/plugin-fusion": "^1.0.4",
    "@ice/runtime": "^1.2.7",
    "moment": "^2.29.4",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "styled-components": "^6.0.8"
  },
  "devDependencies": {
    "@applint/spec": "^1.2.3",
    "@ice/app": "^3.3.0",
    "@types/node": "^18.11.17",
    "@types/react": "^18.0.0",
    "@types/react-dom": "^18.0.0",
    "eslint": "^8.35.0",
    "stylelint": "^15.2.0",
    "typescript": "^4.9.5"
  },
  "scripts": {
    "start": "ice start --speedup",
    "build": "ice build",
    "eslint": "eslint ./src --cache --ext .js,.jsx,.ts,.tsx",
    "eslint:fix": "npm run eslint -- --fix",
    "stylelint": "stylelint \"src/**/*.{css,scss,less}\" --cache",
    "stylelint:fix": "npm run stylelint -- --fix"
  },
  "publishConfig": {
    "access": "public"
  },
  "repository": "git@github.com:ice-lab/react-materials.git"
}

**Comment by zhiwei55** [comment]
we just got same issue.some one can reply ?

**Comment by ClarkXia** [comment]
Modify the scripts in package.json:

```diff
- "start": "ice start --speedup",
+ "start": "ice start",
```

We will fix this issue in next bug fix version (Probably publish in 10.12).

**Comment by lilithyang2018** [comment]
> Modify the scripts in package.json:
> 
> ```diff
> - "start": "ice start --speedup",
> + "start": "ice start",
> ```
> 
> We will solve this issue in next bug fix version (Probably publish in 10.12).

去掉speedup参数后启动发现启动错误，打印iconFile之后发现可能是路径问题
![err](https://github.com/alibaba/ice/assets/39897706/64ea3e42-f622-43ac-b88b-19a6de861db6)
把iconFile的值替换为`${themePackage}/icons.scss`之后能正常启动 但是只有按需加载的蓝色样式，主题还是没有效果
![fusion button 3](https://github.com/alibaba/ice/assets/39897706/75b76306-3278-4517-bee7-06efe33c67cf)

**Comment by ClarkXia** [comment]
@lilithyang2018 https://v3.ice.work/docs/guide/advanced/fusion/#%E6%8E%A8%E8%8D%90%E9%85%8D%E7%BD%AE 看下推荐配置

**Comment by ClarkXia** [comment]
win32 下兼容问题 同样会在本周版本修复

**Comment by lilithyang2018** [comment]
> @lilithyang2018 https://v3.ice.work/docs/guide/advanced/fusion/#%E6%8E%A8%E8%8D%90%E9%85%8D%E7%BD%AE 看下推荐配置

推荐配置可以正常显示主题效果，就是importStyle必须是'sass'而不能是true

**Comment by ClarkXia** [comment]
fixed in @ice/plugin-fusion@1.0.5
